.github/workflows/release.yml

```diff
@@ -0,0 +1,143 @@
+name: release
+
+on:
+  push:
+    branches: [ main ]
+    tags:
```

Comment:
The tag CI isn't trigger, maybe we need to write it as 

```yaml
on:
   push:
      branches: [ main ]
      tags: '*'
```

Reply:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Reply:
Yes, can write it as `tags: [ 'roapi-http-v*' ]`? I am planning to release a columnq binary from this repo in the future for pure CLI usage as well. And possibly `roapi-grpc`. So would be good to namespace tags by sub project names.

something like this: https://github.com/delta-io/delta-rs/blob/main/.github/workflows/python_release.yml#L5